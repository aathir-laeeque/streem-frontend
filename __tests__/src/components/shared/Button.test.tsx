import { Button, FlatButton } from '#components';
import React from 'react';
import { create, ReactTestRenderer } from 'react-test-renderer';

describe('<Button />', () => {
  let snapshot: ReactTestRenderer;

  beforeAll(() => {
    const component = (
      <div>
        <Button onClick={() => jest.fn()}>Button</Button>
        <FlatButton onClick={() => jest.fn()}>FlatButton</FlatButton>
      </div>
    );
    snapshot = create(component);
  });

  it('should render and match the snapshot', () => {
    expect(snapshot.toJSON()).toMatchSnapshot();
  });
});
