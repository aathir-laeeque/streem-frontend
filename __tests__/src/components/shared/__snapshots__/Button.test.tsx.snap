// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render and match the snapshot 1`] = `
<div>
  <button
    className="sc-fzokOt gDhfqG"
    onClick={[Function]}
  >
    Button
  </button>
  <button
    className="sc-fzqBZW cHkWYl"
    onClick={[Function]}
  >
    FlatButton
  </button>
</div>
`;
